Libraries/JavaScriptAppEngine/polyfills/document.js

```diff
@@ -1,34 +1,32 @@
-/* eslint global-strict: 0 */
-(function(GLOBAL) {
-  /**
-   * The document must be shimmed before anything else that might define the
-   * `ExecutionEnvironment` module (which checks for `document.createElement`).
-   */
+/* eslint strict: 0 */
```

Comment:
`global-strict` was replaced with `strict` in eslint 1.0 so it was causing some lint warnings.